packages/starknet-snap/src/rpcs/executeTxn.test.ts

```diff
@@ -90,41 +90,53 @@ const prepareMockExecuteTxn = async (
 describe('ExecuteTxn', () => {
   let callsExample: any;
 
-  it('executes transaction correctly if the account is deployed', async () => {
-    callsExample = callsExamples[0];
-    const {
-      account,
-      createAccountSpy,
-      executeTxnRespMock,
-      getEstimatedFeesSpy,
-      getEstimatedFeesRepsMock,
-      request,
-    } = await prepareMockExecuteTxn(
-      callsExample.hash,
-      callsExample.calls,
-      callsExample.details,
-      true,
-    );
+  it.each([
+    {
+      calls: callsExamples[0],
+      callStr: 'Call[]', // This is 'callStr'
+    },
+    {
+      calls: callsExamples[1],
+      callStr: 'Call', // This is 'callStr'
+    },
+  ])(
+    'executes transaction correctly if the account is deployed and calls is $callStr',
+    async ({ calls }: { calls: any }) => {
+      callsExample = calls;
```

Comment:
btw what is the reason to have a var refer to the inputs? 
the test suppose to be stateless